components/Counters/Counters.js

```diff
@@ -42,7 +49,26 @@ function Counters({ boxes, buckets, setBoxes, setBuckets }) {
           />
         ))}
       </CountersContainer>
-    </div>
+      <CountersContainer>
+        {attachments.map((value, index) => (
+          <Counter
+            key={index}
```

Comment:
This may lead to bugs: Now you have three counters on your site, each with key="0" and three counters with key="1" because the other counters (Eimer und Kisten) work in the same way.